.github/workflows/dashboard.yaml

```diff
@@ -13,4 +13,43 @@ jobs:
       with:
         java-version: 8
     - run: java -version
-    - run: .kokoro/dashboard.sh
\ No newline at end of file
+    - run: .kokoro/dashboard.sh
+
+  dependency-convergence-check:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'googleapis' && github.head_ref == 'release-please/branches/master'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+        script: |
```

Comment:
This script setup is used by the `actions/github-script` action, not `setup-java`. In any case, this is probably unnecessary as the `if` clause for the entire job is good enough for deciding whether or not to run the script.